apis/database/v1alpha2/sql_types.go

```diff
@@ -224,23 +292,6 @@ type VirtualNetworkRuleProperties struct {
 	IgnoreMissingVnetServiceEndpoint bool `json:"ignoreMissingVnetServiceEndpoint,omitempty"`
```

Comment:
Do we want a `VirtualNetworkSubnetIdRef` field in `VirtualNetworkRuleProperties`?

Reply:
We do. fixed in #39